zuul-core/src/main/java/com/netflix/zuul/exception/ZuulException.java

```diff
@@ -38,6 +38,17 @@ public ZuulException(String sMessage, Throwable throwable, String errorCause) {
         this.errorCause = errorCause;
     }
 
+    /**
+     * Source Throwable, message, status code and info about the cause
+     * @param sMessage
+     * @param throwable
+     * @param errorCause
+     */
```

Comment:
missing param :)

Reply:
Would it be better if I removed all the @param's? Do they really add necessary documentation?

Reply:
You can get rid of the params but describe the method better.